File: 1_topicos_iniciais/python/read-json/read-file.py
import json
from dateutil import parser

with open('file.json', 'r') as file :
    jsonData = json.load(file)

def parse_timestemp(timestemp):
    return parser.isoparse(timestemp)


for course in jsonData:
    print(f"Curso: {course['title']}")

    for lesson in course['lessons']:
        print(f"    Aula {lesson['id']} : {lesson['title']}")
        print(f"        Media: {lesson['media']}")
        print(f"        Timestamp: {parse_timestemp(lesson['timestamp'])}")
        

    print()
    print()

File: 1_topicos_iniciais/javascript/read-json/js/read-file.js
import { jsonData } from "./file.js";

const parseTimestemp = (timestemp) => {
    return new Date(timestemp);
}

for (const course of jsonData){
    console.log(`Curso: ${course.id} - ${course.title}`)

    for(const lesson of course.lessons){
        console.log(`   Aula ${lesson.id}: - ${lesson.title}`)
        console.log(`       Media: ${lesson.media}`)
        console.log(`       Timestemp: ${parseTimestemp(lesson.timestamp)}`)
    }
}